### ISSUE TITLE
pip install doesn't install latest revision

### ISSUE THREAD

**Issue by hsenot** [jkal/python-aba#1]
Hey John,

Thanks for making this library available, very useful :+1: 

A small issue I encountered was that the basic pip install doesn't install the "latest" version and the total in the file footer is therefore not calculatd properly. Instead, I just uninstalled the aba package and added:
https://github.com/jkal/python-aba/archive/master.zip
in my requirement file instead.

Hope this helps someone facing the same issue.
Cheers, and thanks again,

Herve

**Comment by michaelsproul** [comment]
Yeah I have the same issue and resorted to making a 0.2.1 release on my fork

**Comment by matblas** [comment]
+1 over here, this has been really useful and it would be good to get the latest release on pip for keeping track in requirements files.
